learntools/machine_learning/nbs/tut1.ipynb

```diff
@@ -0,0 +1,78 @@
+    "\n",
+    "Machine learning works the same way.  We'll start with a model called the Decision Tree. There are fancier models that give more accurate predictions. But decision trees are easy to understand, and they are the basic building block for some of the best models in data science.\n",
+    "\n",
+    "For simplicity, we'll start with the simplest possible decision tree. \n",
+    "\n",
+    "![First Decision Trees](http://i.imgur.com/7tsb5b1.png)\n",
+    "\n",
+    "It divides houses into only two categories. You predict the price of a new house by finding out which category it's in, and the prediction is the historical average price from that category.\n",
+    "\n",
+    "This captures the relationship between house size and price. We use data to decide how to break the houses into two groups,  and then again to determine the predicted price in each group.  This step of capturing patterns from data is called **fitting** or **training** the model. The data used to **fit** the model is called the **training data**.  \n",
+    "\n",
+    "The details of how the model is fit (e.g. how to split up the data) is complex enough that we will save it for later. After the model has been fit, you can apply it to new data to **predict** prices of additional homes.\n",
+    "\n",
+    "---\n",
+    "# Example\n",
+    "Which of the following two decisions trees is more likely to result from fitting the real estate training data?\n",
+    "\n",
+    "![First Decision Trees](http://i.imgur.com/prAjgku.png)\n",
+    "\n",
+    "---\n",
+    "# Improving the Decision Tree\n",
+    "\n",
+    "The decision tree on the left (Decision Tree 1) probably makes more sense, because it captures the reality that houses with more bedrooms tend to sell at higher prices than houses with fewer bedrooms.  The biggest shortcoming of this model is that it doesn't capture most factors affecting home price, like number of bathrooms, lot size, location, etc. \n",
+    "\n",
+    "You can capture more factors using a tree that has more \"splits.\" These are called \"deeper\" trees. A decision tree that also considers the total size of each house's lot might look like this: \n",
+    "![Depth 2 Tree](http://i.imgur.com/R3ywQsR.png)\n",
+    "\n",
+    "You predict the price of any house by tracing through the decision tree, always picking the path corresponding to that house's characteristics. The predicted price for the house is at the bottom of the tree.  The point at the bottom where we make a prediction is called a **leaf.**   \n",
+    "\n",
+    "The splits and values at the leaves will be determined by the data, so it's time for you to check out the data you will be working with.\n",
+    "\n",
+    "# Continue\n",
+    "You will write code as part of an ongoing data science project for the rest of the tutorials. **[Click here](https://www.kaggle.com/dansbecker/starting-your-ml-project)** to get started."
```

Comment:
I was a little confused by the verbiage here. It seems to suggest that the next thing we're going to do is write some code.